modules/private/private.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Module Name: Private site template
+ *
+ * @package Jetpack
+ */
+
+nocache_headers();
+header( 'Content-Type: ' . get_bloginfo( 'html_type' ) . '; charset=' . get_bloginfo( 'charset' ) );
+
+$redirect_to = 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
+?><!DOCTYPE html>
+<!--[if IE 8]>
+	<html xmlns="http://www.w3.org/1999/xhtml" class="ie8" <?php language_attributes(); ?>>
+<![endif]-->
+<!--[if !(IE 8) ]><!-->
+	<html xmlns="http://www.w3.org/1999/xhtml" <?php language_attributes(); ?>>
+<!--<![endif]-->
+<head>
+<meta http-equiv="Content-Type" content="<?php bloginfo( 'html_type' ); ?>; charset=<?php bloginfo( 'charset' ); ?>" />
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title><?php echo bloginfo( 'name' ); ?></title>
+<?php
+	// Use styles from wp-login.
+	wp_enqueue_style( 'login' );
+	do_action( 'login_enqueue_scripts' );
+	do_action( 'login_head' );
+
+	$classes = array( 'wp-core-ui' );
+if ( is_rtl() ) {
+	$classes[] = 'rtl';
+}
+	$classes[] = ' locale-' . sanitize_html_class( strtolower( str_replace( '_', '-', get_locale() ) ) );
+
+	$login_link = site_url() . '/wp-login.php?redirect_to=' . $redirect_to;
+?>
+
+</head>
+<body class="login <?php echo esc_attr( implode( ' ', $classes ) ); ?>">
+	<div id="login">
+		<h1><a href="<?php echo esc_attr( $login_link ); ?>" tabindex="-1"><?php bloginfo( 'name' ); ?></a></h1>
+		<div class="message" style="overflow: auto;">
```

Comment:
I wonder if it would make sense to allow site owners to customize that page a bit,  either by offering them a filter to customize the content, or by surfacing some options in the Customizer?

Reply:
I left the Custom CSS in so they could use that. Ultimately we'll be giving users a "coming soon" type landing page, but I'll add that in a different PR.